### ISSUE TITLE
Breadcumb not refreshing when reload the page state internally using shouldReuseRoute as false

### ISSUE THREAD

**Issue by ashokkumarjuly** [udayvunnam/xng-breadcrumb#27]
At First thankyou for the wonderfull plugin,

**Describe the bug**
I have a company dropdown on the header component which is placed inside layout component,if any change in that dropdown, i have to reload the widgets and table data to fetch the company related datas,
So instead of hard reloading the page,  we using  the below code on dropdown change
	this.router.routeReuseStrategy.shouldReuseRoute = function() {
		return false;
	};
	const currentUrl = this.router.url;
		this.router.navigateByUrl(currentUrl).then(() => {
		this.router.navigated = false;         
		this.router.navigate([currentUrl]);
	});

then breadcumb is not updating and showing the old values, even when i navigate to any pages.
 
But if i browser reload the whole page, breadcumb working properly, the issue only occurs after header dropdown change.
 
I have been stuck on this breadcumb issue, please help.

**Comment by udayvunnam** [comment]
@ashokkumarjuly This has been lost in a while, are you still seeing the issue?

**Comment by brian-teamcoda** [comment]
I've also ran into issues using the Angular's routing function routeReuseStrategy (similar to the example). Breadcrumbs only update on page refresh, but not on navigation anymore. Using version 6.1.0.

Problems stops occurring when I comment out    
```
this.router.routeReuseStrategy.shouldReuseRoute = () => {
   return false;
};
```

I need the code to update pages correctly within my project.

**Comment by udayvunnam** [comment]
@brian-teamcoda I will take a look and provide possible solution

**Comment by jj-whitehead** [comment]
> I've also ran into issues using the Angular's routing function routeReuseStrategy (similar to the example). Breadcrumbs only update on page refresh, but not on navigation anymore. Using version 6.1.0.
> 
> Problems stops occurring when I comment out
> 
> ```
> this.router.routeReuseStrategy.shouldReuseRoute = () => {
>    return false;
> };
> ```
> 
> I need the code to update pages correctly within my project.

Getting the same issue in our solution. Any updates on this @udayvunnam ?

**Comment by udayvunnam** [comment]
I am on vacation currently. Will try to fix this ASAP. Thanks for reporting the issue.

**Comment by udayvunnam** [comment]
@ashokkumarjuly  @jj-whitehead @brian-teamcoda This is fixed in the latest version 6.3.3. Thanks for reporting the issue.

**Comment by AkmaralB** [comment]
Hello, @udayvunnam ! Thanks for the plugin. However, I am getting the same issue with Angular v11.0.3, xng-breadcrumb v6.5.0. Could you have a look?

**Comment by udayvunnam** [comment]
@AkmaralB the last time I checked it was working. Can you please provide a reproducible example

**Comment by AkmaralB** [comment]
> @AkmaralB the last time I checked it was working. Can you please provide a reproducible example

[angular-tour-of-heroes.zip](https://github.com/udayvunnam/xng-breadcrumb/files/5799777/angular-tour-of-heroes.zip)

In project in file cashbox-registration.component.ts we using shouldReuseRoute function. If open this page(/cabinet/cashboxes/new) and then navigate to other pages xng-breadcrumbs doesn't display active links and only display link of problem page. But without shouldReuseRoute function (comment lines) xng-breadcrumbs working properly.

**Comment by udayvunnam** [comment]
@ashokkumarjuly @jj-whitehead @brian-teamcoda  @AkmaralB

Due to a bug in Angular, this issue persisted. Now fixed again with **xng-breadcrumb@6.8.0**. Please upgrade to latest version of xng-breadcrumb
